tee-apps/sealed-bid-auction/src/api/AuctionApiServer.ts

```diff
@@ -38,7 +38,10 @@ export class AuctionApiServer {
             } : {},
             routes: {
                 "/healthcheck": new Response("OK"),
-                "/api/preauction": req => this.auctionController.preauction(req),
+                "/api/preauction": {
+                    GET: _req => new Response("Use POST", {status: 405}),
```

Comment:
The 405 response should include CORS headers, otherwise the browser might block the error. Can include include `Allow` to show supported methods to users

```suggestion
                    GET: _req => new Response("Use POST", {
                      status: 405,
                      headers: { ...CORS_HEADERS, "Allow": "POST, OPTIONS" },
})
```

Reply:
On top of that, browsers preflight `POST` with JSON/custom headers. Please add an `OPTIONS` handler so the request isn’t blocked by CORS. Also make sure these responses carry CORS headers.